psyslog.js

```diff
@@ -23,21 +49,9 @@ if (args.config) {
   } catch (e) {
     process.stderr.write(`could not load settings file, using defaults: ${e.message}`)
   }
+  options = Object.assign(options, userOptions)
```

Comment:
We clearly had a bug previously that has been replicated here. It should be:

```suggestion
  options = Object.assign({}, options, userOptions)
```

Reply:
Fixed in new commit.